package.json

```diff
@@ -30,7 +30,7 @@
     "html-encoding-sniffer": "^1.0.2",
     "left-pad": "^1.2.0",
     "nwmatcher": "^1.4.3",
-    "parse5": "^4.0.0",
+    "parse5": "4.0.0",
```

Comment:
Not sure how Yarn would resolve the difference here, but you might want to update the lockfile as well to be sure.